### DISCUSSION TITLE
Need help: Missing education benefits after 5 days

### DISCUSSION THREAD

**Discussion by Andrew-star-code** [community/community#132753] [GitHub Education]
### Hi everyone,

It's been 5 days since approval, and it still is saying that my benefits will be soon available.
Do I need to do something?

**Comment by davevad93** [comment]
Hi @Andrew-star-code ,

It usually takes a few days, but sometimes it can even take a couple of weeks; the waiting time may vary depending on different factors, e.g. the volume of applications.

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @Andrew-star-code was approved and your benefits are now available.`
